integration_tests/src/main/python/spark_session.py

```diff
@@ -189,3 +189,13 @@ def get_java_major_version():
     elif dash_pos != -1:
         ver = ver[0:dash_pos]
     return int(ver)
+
+def get_jvm_charset():
+    sc = SparkContext.getOrCreate()
```

Comment:
could reuse _spark from L30
```suggestion
    sc = _spark.sparkContext
```